docker-compose.yml

```diff
@@ -28,6 +28,7 @@ x-robotoff-base-env: &robotoff-base-env
   OFF_PASSWORD:
   INFLUXDB_HOST:
   INFLUXDB_PORT:
+  INFLUX_DB_NAME:
```

Comment:
This should be `INFLUXDB_NAME`

Reply:
seems I was tired while writing this :sweat_smile: